test/controllers/project_controller_test.exs

```diff
@@ -0,0 +1,58 @@
+defmodule CodeCorps.ProjectControllerTest do
+  use CodeCorps.ConnCase
+
+  alias CodeCorps.Project
+
+  setup do
+    conn =
+      %{build_conn | host: "api."}
+      |> put_req_header("accept", "application/vnd.api+json")
+      |> put_req_header("content-type", "application/vnd.api+json")
+
+    {:ok, conn: conn}
+  end
+
+  defp build_payload(id, attributes, relationships \\ %{}) do
+    build_payload(attributes)
+    |> put_in(["data", "id"], id)
+    |> put_in(["data", "relationships"], relationships)
+  end
+
+  defp build_payload(attributes) do
+    %{
+      "data" => %{
+        "type" => "category",
+        "attributes" => attributes
+      }
+    }
+  end
+
+  test "lists all entries on index", %{conn: conn} do
+    conn = get conn, project_path(conn, :index)
+    assert json_response(conn, 200)["data"] == []
+  end
+
+  test "lists all entries for organization specified by slug"
+
+  test "shows chosen resource", %{conn: conn} do
+    organization = insert_organization
+    project = insert_project(%{
+      title: "Test project",
+      description: "Test project description",
+      long_description_markdown: "A markdown **description**",
+      organization: organization})
+
+    conn = get conn, project_path(conn, :show, project)
+    data = json_response(conn, 200)["data"]
+    assert data["id"] == "#{project.id}"
+    assert data["type"] == "project"
+    assert data["attributes"]["title"] == "Test project"
+    assert data["attributes"]["description"] == "Test project description"
+    assert data["attributes"]["long-description-markdown"] == "A markdown **description**"
+    assert data["relationships"]["organizations"]["id"] == organization.id
+  end
+
+  test "shows chosen resource retrieved by slug"
```

Comment:
Do we want to retrieve this by `slug`? Is it needed? I think I missed your points on this re: the Ember app before, but not positive now. Do we still need it?

I would like to avoid the same API structure we currently have if possible. The routing was really weird and I think not necessary in a number of cases.